src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -247,6 +247,11 @@ public bool Explicit
         /// </summary>
         public string? ExcludePlatform { get; set; }
 
+        /// <summary>
+        /// Comma-delimited list of platforms to run the test for
```

Comment:
Note to self: Fix this copy/paste error

Reply:
Yes, copy it from TestFixtureAttribute's TypeArgs property instead.